sgpt/app.py

```diff
@@ -113,7 +125,7 @@ def main(  # pylint: disable=too-many-arguments,too-many-locals
 
     if repl:
         # Will be in infinite loop here until user exits with Ctrl+C.
-        ReplHandler(client, repl, shell, code).handle(
+        ReplHandler(client, repl, shell, describe_shell, code).handle(
```

Comment:
It doesn't really work in REPL mode.
```
>>>sgpt --repl temp --describe-shell
Entering REPL mode, press Ctrl+C to exit.
>>> ls -la
ls -la
>>> echo hello world
echo hello world
```

Reply:
I believe it would be easier to fix it with roles from main, once the branch will be rebased.